Makefile

```diff
@@ -63,7 +63,8 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
+	$(YARN) check
```

Comment:
That's Make telling you that the dependencies of the `setup` target have not changed, so there's no work to be done.

```make
.PHONY: foo
foo: bar

bar:
	echo hello >'$@'
```

```console
$ make foo
echo hello >'bar'

$ make foo
make: Nothing to be done for `foo'.
```